tf_mat_base_link1_0:
  target_pose:
    position: [0.072125, 0.072125, 0.018]
    orientation:
      - [-2.597348e-06, -0.7071068, 0.7071068]
      - [-2.597348e-06, -0.7071068, -0.7071068]
      - [1, -3.673205e-06, -2.220446e-16]
tf_mat_base_link4_0:
  target_pose:
    position: [0.072125, -0.072125, 0.018]
    orientation:
      - [-2.597348e-06, -0.7071068, -0.7071068]
      - [2.597348e-06, 0.7071068, -0.7071068]
      - [1, -3.673205e-06, 2.220446e-16]
tf_mat_flan1_flan4:
  target_pose:
    position: [-0.08926057, 1.059978, -0.001109014]
    orientation:
      - [0.9967778, -0.07062707, -0.03802515]
      - [0.07060583, 0.9975025, -0.001902748]
      - [0.03806457, -0.0007881802, 0.999275]
tf_mat_base_link2_0:
  target_pose:
    position: [-0.072125, 0.072125, 0.018]
    orientation:
      - [2.597348e-06, 0.7071068, 0.7071068]
      - [-2.597348e-06, -0.7071068, 0.7071068]
      - [1, -3.673205e-06, 5.551115e-16]
tf_mat_base_link3_0:
  target_pose:
    position: [-0.072125, -0.072125, 0.018]
    orientation:
      - [2.597348e-06, 0.7071068, -0.7071068]
      - [2.597348e-06, 0.7071068, 0.7071068]
      - [1, -3.673205e-06, -2.220446e-16]